railgun/test/integration/ingress_test.go

```diff
@@ -45,6 +46,15 @@ func TestMinimalIngress(t *testing.T) {
 	ingress, err = cluster.Client().NetworkingV1().Ingresses("default").Create(ctx, ingress, metav1.CreateOptions{})
 	assert.NoError(t, err)
 
+	// ensure cleanup of the ingress
+	defer func() {
+		if err := cluster.Client().NetworkingV1().Ingresses("default").Delete(ctx, ingress.Name, metav1.DeleteOptions{}); err != nil {
+			if !errors.IsNotFound(err) {
+				assert.NoError(t, err)
```

Comment:
Would `require` be better than `assert` in these cases?

Reply:
2ec6d7fef59b9a89f768eccfe9a730931871907e